index.html

```diff
@@ -7526,6 +7526,27 @@ <h5>Protocol and Payload Bindings</h5>
           </p>
         </section>
       </section>
+
+      <section id="profile-relationship">
+        <h4>Relationship to Profiles</h4>
+        <p>
+          The Web of Things also define a profiling mechanism in [[WOT-PROFILES]] that allows the reduction of
+          implementation effort. Profiles are built on top of existing Bindings by constraining the flexibility of TDs.
+          Such constraints can be bindings, semantic contexts, link relations, security schemes and discovery
+          mechanisms. As there is less flexibility, the implementation effort to achieve out-of-the-box interoperability
+          is reduced.
+        </p>
+        <p>
+          For example, a profile can constrain the usage of HTTP GET Method for <code>readproperty</code> operations and
+          require payloads to be always serialized in JSON. The usage of a profile is indicated by the
+          <code>profile</code> term in the TD, which is a URI that points to the profile definition.
+          <span class="rfc2119-assertion" id="td-consumer-profile-ignore">
+            A Consumer MAY ignore the <code>profile</code> term and use the TD as it is, but the knowledge of a profile
+            allows the Consumer to implement the profile-specific constraints as assumptions and thus reduce the
+            implementation effort on understanding the TD.
+          </span>
```

Comment:
I agree with this assertion, but I'm not sure it belongs in the Bindings section of the specification, since the constraints of Profiles extend beyond bindings.

Reply:
> How about referencing https://w3c.github.io/wot-profile/#profiling-mechanism ?

That is probably the best we can do for now.

I do think the assertion seems a bit out of place here though.

Reply:
Done :)